Sources/MuxUploadSDK/PublicAPI/MuxUpload.swift

```diff
@@ -48,7 +48,7 @@ public final class MuxUpload : Hashable, Equatable {
  
     private let uploadInfo: UploadInfo
     private let manageBySDK: Bool
-    private let id: String = UUID().uuidString
+    private var id: String = UUID().uuidString
```

Comment:
It's possible (probably desirable) to keep the `id` immutable.

The only path to initializing it is by the `UploadInfo` input (which is good), so we don't need a backup value
```suggestion
        private let id: String
```

Reply:
https://github.com/muxinc/swift-upload-sdk/pull/31/commits/a2c060483427388d851bd3726c96ec58daf5710e